### PR TITLE
Fix footer

### PR DESCRIPTION
**Pull request by bryantgillespie**

### CODE REVIEW

File: components/Nav/Footer.vue [hunk 1]
```diff
@@ -37,7 +38,7 @@ const year = new Date().getFullYear();
 const socials = {
 	discord: 'https://directus.chat',
 	docker: 'https://hub.docker.com/r/directus/directus',
-	twitter: 'https://twitter.com/directus',
+	x: 'https://x.com/directus',
```
Comment (rijkvanzanten) [inline components/Nav/Footer.vue:41]: I don't think I'll ever get used to this 😂

Comment (bryantgillespie) [reply components/Nav/Footer.vue:41]: leave it for nostalgia?

Comment (bryantgillespie) [reply components/Nav/Footer.vue:41]: not sure if we hardcoded it anywhere else

Comment (bryantgillespie) [reply components/Nav/Footer.vue:41]: ResourcePage maybe?

Comment (rijkvanzanten) [reply components/Nav/Footer.vue:44]: I'll handle those 👍🏻

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**